r2/example.ini

```diff
@@ -159,6 +159,11 @@ karma_to_vote_in_overview = 1000
 karma_percentage_to_be_voted = 80
 poll_max_choices = 20
 
+# If user's karma is less than threshold, thing.votable = False
+# meaning that the voting icons won't appear on posts and comments
+# "karma_to_vote" is only used for voting in polls.
+no_voting_threshold = 10
```

Comment:
I think this setting should be renamed to remove the negative `no` and be clearer about what the value is. The best way to do this may be to rename `karma_to_vote` to `karma_to_vote_on_polls` and make this setting `karma_to_vote`.